### PR TITLE
Chapter 3 of Federal Field Guide / Doing the work

### PR DESCRIPTION
**Pull request by cmajel**
This PR adds content from Chapter 3 of the Federal Field Guide ([design google doc](**url**)). It also includes some styling adjustments to headings to account for more nested headings in the Federal Field Guide:

Changes include:
* Converted Chapter 3 content to markdown
* Added Appendix page placeholder 
* Added shortened titles for sidenav
* Sized up all headings by one USWDS size token (eg `sm` >> `md) and added a color differentiator and consistent text styles to `h5` and `h6`, as this is the first time we encountered these in this guide. As a consequence of this, did a few manual adjustments to other sections of the guide to keep sizing consistent with the current design. 

### Example
`Recommendations` is a `h4`, `How to oversee` is a `h5` and Sprint Reviews is a `h6`

#### Before
1 before

#### After
1 after

### CODE REVIEW

File: _pages/federal-fieldguide/doing-the-work.md [hunk 1]
```diff
@@ -4,9 +4,311 @@ title: Federal field guide
...
+A thoughtful post-award kick-off meeting should introduce new team members, empower the government product owner, plan next steps, and get the team excited about the upcoming work.
+
+#### Recommendations
+
+[FAR Part 42.5](https://www.acquisition.gov/content/part-42-contract-administration-and-audit-services#i1075565) encourages Contracting Officers (COs) to hold a post-award kick-off meeting with the winning contractor. That is often treated somewhere between a check-box exercise and an afterthought.
+
+We recommend being more intentional about the kick-offs, either as one combined meeting (contractual + project) or as two separate meetings scheduled in close proximity to one another. Some Contracting Officers have strong feelings about their procedures for a FAR-required kickoff, and prefer to hold their own separate meeting.
+
+Goals of a post-award kick-off:
+
```
Comment (igorkorenfeld) [inline _pages/federal-fieldguide/doing-the-work.md:44]: May make these into regular paragraphs instead of lists

**Review by igorkorenfeld — COMMENTED** [review]
This is looking great just a few things I'd want to discuss
1. The new heading sizes work great, but the h6 is a little weird to me in that it's smaller than the body text below it. Two ideas for how to change: 
a. keep it as `sm` but make it `transform: uppercase`
b. make it `md` but `font-weight:light` (or whatever the actual USWDS equivalent actually is). 

2. I think it works that the "challenge" sections match the print book , but now I am wondering if it's weird that the "Download this guide" note on the State guide is styled the same way? We should probably rename the class to something more semantic if we do use it in both spots (or just for the challenges. 

3. Some of the bullet lists can just be paragraphs. I made an in-line comment as an example of one. Basically any time there is bold text, I think that styling is enough and a bullet list might be unnecessary. Nothing for you to change there, I am mostly flagging so I remember to take a look at it later.

File: _pages/federal-fieldguide/doing-the-work.md (hunk 1 above)
Comment (cmajel) [reply _pages/federal-fieldguide/doing-the-work.md:44]: cool, they're rendered as paragraphs now - preview below 

Screen Shot 2020-09-03 at 9 20 17 AM

**Comment by cmajel** [comment]
@igorkorenfeld thanks for the review! A few thoughts on the points you made:

1. I was looking again at the `h6` styling and realized it was set to `xs` and no headings had been set to `sm`. So I tried sizing it up. First, here's what it looks like now. `Sprint reviews` is the `h6`.
2 before


a. Here's what it looks like set as `sm` which puts it the same size as the body copy (but bold and a different color). 
2 after

b. I also tried your `b` suggestion to size up to `md` but a lighter font weight:
3 after

I'm liking the first option - what do you think?

2. Agreed on the class name. How about I update it to `.guide-highlight`?

3. Ok cool. For now I'll match the styling in the docs, but happy to revisit this with you once we have it in and look and what needs adjusting.

**Comment by igorkorenfeld** [comment]
Nice, thank you for making these adjustments and putting in all these screenshots in. Very helpful!

1. I think I actually lean towards option b, because I think if you don't pay enough attention `h6` and `h5` can look very similar with both being the same color and weight. For me, the shift in weight helps create more distinction and reinforce that the text under `h6` is within the section marked by `h5` 

2. That works great!

3. Perfect.

**Comment by cmajel** [comment]
@igorkorenfeld updates made 👍

**Comment by igorkorenfeld** [comment]
This looks good, thank you!

**Pull request merged**